tsconfig.build.json

```diff
@@ -1,10 +1,9 @@
 {
-  "extends": "./tsconfig.json",
-  "compilerOptions": {
-    "declaration": true,
-    "declarationDir": "./dist",
-    "outDir": "./dist",
-    "rootDir": "./src"
-  },
-  "exclude": ["**/*.test.ts"]
+  "references": [
+    { "path": "./packages/base-controller/tsconfig.build.json" },
```

Comment:
We must be careful, when adding new packages to this file, to use the `tsconfig.build.json` versions of all packages, otherwise test files will get included in the build (and the built files will get put in the wrong place).